### PR TITLE
Fixing issue 162 and one more bug and increase the test coverage

### PR DESCRIPTION
**Pull request by emitreva-cb**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Tests have been added that prove the fix is effective or that the feature works.
- [x] New and existing tests pass locally with the changes.
- [x] Code follows the style guidelines of this project (PEP8, clean code, linter).
- [x] A self-review of the code has been done.

## What is the ticket or issue number?
https://jira.carbonblack.local/browse/CBAPI-2567
https://github.com/carbonblack/carbon-black-cloud-sdk-python/issues/162

## Does this introduce a breaking change?

- [x] Yes
- [ ] No

download_url is no longer a property, but a method, so the proper way to get the dowload_url will be either file_pull.download_url(expiration_seconds=30) or file_pull.download_url()

## How Has This Been Tested?
With a python script.

### CODE REVIEW

**Review by avanbrunt-cb — APPROVED** [review]
Looks good. My one thought is we should add something to read the docs indicating that a `NonQueryableModel` error will still support the select by id but the search capability

**Comment by emitreva-cb** [comment]
> Looks good. My one thought is we should add something to read the docs indicating that a `NonQueryableModel` error will still support the select by id but the search capability

Good point, added 

> Looks good. My one thought is we should add something to read the docs indicating that a `NonQueryableModel` error will still support the select by id but the search capability

Good point, I have included a guide how to obtain the download url with couple of ways to do it.

File: docs/unified-binary-store.rst [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+Get Download URL
+----------------
+
+::
+
+    >>> from cbc_sdk import CBCloudAPI
+    >>> api = CBCloudAPI(profile='sample')
+    >>> from cbc_sdk.enterprise_edr.ubs import Binary
+    >>> ha256_hash = '8005557c1614c1e2c89f7db3702199de2b1e4605718fa32ff6ffdb2b41ed3759'
```
Comment (avanbrunt-cb) [inline docs/unified-binary-store.rst:15]: sha256_hash instead of ha256_hash

File: docs/unified-binary-store.rst [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    >>> binary = Binary(cb, ha256_hash)
+    >>> download_url = binary.download_url()
+
+*Note: This is going to generate download link for the binary that will be active for 1 hour (default expiration period).*
```
Comment (avanbrunt-cb) [inline docs/unified-binary-store.rst:19]: We should show an example value for what download_url will be

File: docs/unified-binary-store.rst [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+    >>> from cbc_sdk import CBCloudAPI
+    >>> api = CBCloudAPI(profile='sample')
+    >>> from cbc_sdk.enterprise_edr.ubs import Binary
+    >>> ha256_hash = '8005557c1614c1e2c89f7db3702199de2b1e4605718fa32ff6ffdb2b41ed3759'
+    >>> binary = cb.select(Binary, ha256_hash)
+
+*Note: If we try to use* :code:`binary = cb.select(Binary)` *, it will fail with exception that the model is non queryable model.* 
+
+Find a full list of supported commands in the
```
Comment (avanbrunt-cb) [inline docs/unified-binary-store.rst:52]: I'm not sure what you mean by commands? Given this is the CBC SDK would it make more sense to reference the UBS portion of our Read the Docs? https://carbon-black-cloud-python-sdk.readthedocs.io/en/latest/cbc_sdk.enterprise_edr/#module-cbc_sdk.enterprise_edr.ubs

If there are additional API endpoints which we have not included in the CBC SDK then we need to create some stories and get them added

File: docs/unified-binary-store.rst (hunk 1 above)
Comment (avanbrunt-cb) [reply docs/unified-binary-store.rst:15]: Apply in all locations

File: docs/unified-binary-store.rst (hunk 2 above)
Comment (avanbrunt-cb) [reply docs/unified-binary-store.rst:19]: Apply in all locations

**Review by avanbrunt-cb — APPROVED** [review]

**Review by hkaragitliev-cb — APPROVED** [review]

**Review by lhilmes-cb — APPROVED** [review]

**Pull request merged**